utils/gear-runtime-test-cli/src/command.rs

```diff
@@ -249,7 +250,11 @@ macro_rules! command {
                     state: gear_common::ProgramState::Initialized,
                     gas_reservation_map: Default::default(),
                 };
-                ProgramStorageOf::<Runtime>::add_program(ProgramId::from_origin(*id), program);
+                ProgramStorageOf::<Runtime>::add_program(
+                    ProgramId::from_origin(*id),
+                    program,
+                    DEFAULT_BLOCK_NUMBER.into(),
```

Comment:
```suggestion
                    0.into(),
```

Reply:
It's unreadable. Named constant are better than just no name number